### PR TITLE
splash: update to v2.9.0

### PR DESCRIPTION
**Pull request by danieljprice**
* update to version 2.9.0

Closes: https://trac.macports.org/attachment/ticket/58301/

#### Description

I submitted the trac ticket first, but then read that pull requests are preferred. Hence my first attempt, this one should be straightforward.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.4
Xcode 10.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/splash/Portfile [hunk 1]
```diff
@@ -19,8 +19,8 @@ license             GPL-2+
 
 worksrcdir          ${name}
 
-checksums           rmd160  1eef9c352ae507c4c8f233431a909bfc5a7fdc06 \
-                    sha256  de06b159846bd20717a55ec7cc9420e0fd86ab38dc66dc74b1f61b77b4d24fce
+checksums           rmd160  fd91759042a5522f911a709cd2345ca0f2f3be9c \
+                    sha256  a3fa807ce2322421af3719fdc87d9214e6937c5ba004e7d8767d32f9e74d8cf9
```
Comment (mf2k) [inline science/splash/Portfile]: We are now adding the `size` parameter to `checksums`.

**Comment by cjones051073** [comment]
@danieljprice can you please address @mf2k ‘s comments, thanks ?

**Comment by cjones051073** [comment]
Thanks for adding size, but please follow our commit guidelines. You need to squash those two commits into one, and make sure the commit message follows the `portname: explanation` pattern.

**Comment by cjones051073** [comment]
Ping @danieljprice . Can you address the comments please ?

**Comment by cjones051073** [comment]
The PR still has two commits in it, one of which is now a merge commit, which means you did not properly rebase your changes before the last push. can you please address this ? This PR needs to show only *one* commit, your required changes, before it can be merged.

**Comment by cjones051073** [comment]
I have manually squashed and merged this PR myself.

**Pull request merged**